### ISSUE TITLE
Is weight decay required?

### ISSUE THREAD

**Issue by ZhenyF** [jmiller656/EDSR-Tensorflow#27]
For L2 loss we need a weight decay to prevent over fitting.

Is it required to set a weight decay for L1 loss?

**Comment by jmiller656** [comment]
I didn't use any normalization techniques in this implementation because they weren't mentioned in the paper
